packages/deno/src/executors/emit/emit.impl.ts

```diff
@@ -9,6 +9,7 @@ import { dirname, join, resolve } from 'path';
 import { BuildExecutorSchema } from './schema';
 
 import { ensureDirSync, unlinkSync, writeFileSync } from 'fs-extra';
+import { CopyAssetsHandler } from '../../utils/assets/copy-assets-handler';
```

Comment:
I think i'd rather have a deep import on '@nx/js' to use that instead of coping over the code, would make maintanace much easier down the road IMO.